CHANGELOG.md

```diff
@@ -93,14 +104,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 - fix: token capacity alarm message ordering [`43ac12b`](https://github.com/isomerpages/isomercms-backend/commit/43ac12bab0114f7563428ce90cc9f55584a43422)
 
-#### [v0.24.1](https://github.com/isomerpages/isomercms-backend/compare/v0.24.0...v0.24.1)
-
-> 27 April 2023
-
-- fix(review request service): use repo name rather than site name [`a323551`](https://github.com/isomerpages/isomercms-backend/commit/a323551feda7b077a7d5d6c2d11ae841ece9bf8d)
-- fix(test cases): fix failing tests [`c843865`](https://github.com/isomerpages/isomercms-backend/commit/c84386549a28602e1af10cd2eda9f056212afaa7)
-
-#### [v0.24.0](https://github.com/isomerpages/isomercms-backend/compare/v0.23.1...v0.24.0)
+#### [v0.24.1](https://github.com/isomerpages/isomercms-backend/compare/v0.23.1...v0.24.1)
```

Comment:
hey i think these got committed by accident?

Reply:
@alexanderleegs i think need to rebase

Reply:
Yup fixed on rebase!